src/styles/blocks/header.scss

```diff
@@ -0,0 +1,65 @@
+@import 'https://fonts.googleapis.com/css?family=Roboto:500&display=swap';
```

Comment:
Importing fonts directly in a SCSS file is not recommended as it can lead to performance issues. Consider importing the font in the HTML file's <head> section instead, where it can be loaded asynchronously.